docs/developer-docs/latest/developer-resources/cli/snippets/installation-prerequisites.md

```diff
@@ -1,5 +1,5 @@
 The installation requires the following software to be already installed on your computer:
 
-- [Node.js](https://nodejs.org): only LTS versions are supported (v14 and v16). Other versions of Node.js may not be compatible with the latest release of Strapi. The 16.x version is most recommended by Strapi.
+- [Node.js](https://nodejs.org): only LTS versions are supported (v14, v16 and v18). Other versions of Node.js may not be compatible with the latest release of Strapi. The 16.x version is most recommended by Strapi.
```

Comment:
```suggestion
- [Node.js](https://nodejs.org): only maintenance and LTS versions are supported (v14, v16, and v18). Other versions of Node.js may not be compatible with the latest release of Strapi. Node v18.x is recommended for Strapi v4.3.9 and above, Node v16.x for Strapi v4.0.x to v4.3.8.
```

Thank you very much, @rembrandtreyes. I double-checked with our engineering team and this is how it should actually be updated :)